src/Akka.Streams.Kafka.Tests/BugFix240SupervisionStrategy.cs

```diff
@@ -219,7 +219,7 @@ await Source.From(Enumerable.Range(1, 11))
                 Subscriptions.AssignmentWithOffset(new TopicPartitionOffset(topicPartition, Offset.Beginning)))
             .Select(c =>
             {
-                if (++count == 7)
```

Comment:
Had to undo some of my previous "fixes" for dealing with N+1 issues with offset committing